src/lnd-actions/connect-peer.js

```diff
@@ -5,22 +5,26 @@ const { deadline } = require('../grpc-utils')
  * connected
  *
  * @private
- * @param {Error} err
+ * @param {Object} err
+ * @param {number} err.code
+ * @param {string} err.details
  * @returns {boolean}
  */
 function alreadyConnected (err) {
-  return (err && err.code === 2 && err.details && err.details.includes('already connected to peer'))
+  return (err && err.code === 2 && err.details !== undefined && err.details.includes('already connected to peer'))
```

Comment:
a style guide I read a long time ago suggested doing nullish checks in javascript with `!= null`. Here, if `err.details = null`, then `(err.details !== undefined) === true`.

Reply:
unless you are certain `err.details` will never be `null`

Reply:
Hmm, I tried that yesterday and I think it gave me a lint error but now it seems to be ok